Validation/RecoTrack/python/TrackValidation_cff.py

```diff
@@ -622,31 +627,52 @@ def _uniqueFirstLayers(layerList):
                 names = "_selectorsFromPVPt09Standalone", task = "_tracksValidationSelectorsFromPVPt09Standalone")
 
 # MTV instances
-trackValidatorStandalone = trackValidator.clone()
-trackValidatorTPPtLess09Standalone = trackValidatorTPPtLess09.clone()
+trackValidatorStandalone = trackValidator.clone(
+    cores = cms.InputTag("highPtJets")
```

Comment:
Here and below, please remove the explicit type from the right hand side (then the existence of `cores` parameter in the cloned-from object is checked)
```suggestion
    cores = "highPtJets"
```

Reply:
ok done